src/main/java/duke/FindCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+public class FindCommand extends Command {
+
+    private String taskToFind = null;
```

Comment:
Initializing `taskToFind` to `null` might not be necessary, but it is nothing to worry about.